modules/datastore/tests/src/Functional/DictionaryEnforcerTest.php

```diff
@@ -96,8 +98,10 @@ public function setUp(): void {
     $this->webServiceApi = ImportController::create(\Drupal::getContainer());
     $this->datasetStorage = \Drupal::service('dkan.metastore.storage')->getInstance('dataset');
     // Copy resource file to uploads directory.
-    $upload_path = \Drupal::service('file_system')->realpath(self::UPLOAD_LOCATION);
-    \Drupal::service('file_system')->copy(self::TEST_DATA_PATH . self::RESOURCE_FILE, $upload_path);
+    $file_system = \Drupal::service('file_system');
+    $upload_path = $file_system->realpath(self::UPLOAD_LOCATION);
+    $file_system->prepareDirectory($upload_path, FileSystemInterface::CREATE_DIRECTORY);
+    $file_system->copy(self::TEST_DATA_PATH . self::RESOURCE_FILE, $upload_path);
```

Comment:
This should fix some of the intermittent test failures we've had related to the uploaded_resources directory not existing before this test was run.